packages/flutter/test/material/floating_action_button_test.dart

```diff
@@ -1054,6 +1071,291 @@ void main() {
     expect(rawMaterialButton.textStyle, style.copyWith(color: const Color(0xffffffff)));
   });
 
+  group('Material 2', () {
```

Comment:
hard to tell with the diff, but just want to confirm these were just copied without modifications

Reply:
Yeah, they were copied from the original tests. The only change was to add the `useMaterial3: false` theme to each of them. I then changed the existing tests to default to M3 so that when we remove the `useMaterial` flag we can just delete this M2 group and the original tests will be the main tests for this.